api/src/test/java/org/openmrs/module/eptsreports/reporting/mock/calculation/EptsCalculationUtilsMock.java

```diff
@@ -0,0 +1,32 @@
+package org.openmrs.module.eptsreports.reporting.mock.calculation;
+
+import mockit.Mock;
+import mockit.MockUp;
+import org.openmrs.Encounter;
+import org.openmrs.Obs;
+import org.openmrs.calculation.result.CalculationResultMap;
+import org.openmrs.module.eptsreports.reporting.utils.EptsCalculationUtils;
+
+/** Mocks {@link EptsCalculationUtils} with org.jmockit */
+public class EptsCalculationUtilsMock extends MockUp<EptsCalculationUtils> {
```

Comment:
This mock class can be removed - the tests still execute successfully without them